lib/http-bridge/test/integration/Test/Integration/HttpBridge/Scenario/API/Transactions.hs

```diff
@@ -47,6 +48,16 @@ import qualified Network.HTTP.Types.Status as HTTP
 
 spec :: forall t. (EncodeAddress t, DecodeAddress t) => SpecWith (Context t)
 spec = do
+    it "TRANS_EXTERNAL_CREATE_05 - not supported for http-bridge" $ \ctx -> do
```

Comment:
I wonder if it is worth adding anything to http-bridge at this momement :thinking:

Reply:
I had the same hesitation, on the other hand this error message is actually returned so decided to add such test also.